.github/workflows/R-CMD-check.yaml

```
@@ -0,0 +1,51 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
```

```suggestion
    branches: main
```

We can remove the `master` option here as we probably won't use it as default branch name